### PR TITLE
Update README

### LINKED ISSUE #17: Add more descriptive README with information "what does this tool for?"
From the current README it quite to impossible to understand, what kind of linting this tool does. 

I suggest you to add some light description about the possibilities of the tool. So the enduser can understand if he need it for his project or not.

### PR DESCRIPTION
**Pull request by eGavr**
closes #17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,28 @@
...
+    - [bemhint](#bemhint)
+    - [reporters](#reporters)
+    - [Example](#example-1)
 
 <!-- TOC END -->
 
 ## Status
 
-The initial stage of development. Code is unstable. Your [suggestions and comments](https://github.com/eGavr/bemhint/issues/new) are wellcome.
+The initial stage of development. Code is unstable. Your [suggestions and comments](https://github.com/bem/bemhint/issues/new) are wellcome.
```
Comment (zxqfox) [inline README.md]: ouch. double-l in `welcome`.

File: README.md [hunk 2]
```diff
@@ -1,22 +1,28 @@
-# BEM hint [![Build Status](https://travis-ci.org/eGavr/bemhint.svg)](https://travis-ci.org/eGavr/bemhint) [![Coverage Status](https://img.shields.io/coveralls/eGavr/bemhint.svg)](https://coveralls.io/r/eGavr/bemhint?branch=master) [![Dependency Status](https://david-dm.org/eGavr/bemhint.svg)](https://david-dm.org/eGavr/bemhint) [![devDependency Status](https://david-dm.org/eGavr/bemhint/dev-status.svg)](https://david-dm.org/eGavr/bemhint#info=devDependencies)
+# BEM hint [![Build Status](https://travis-ci.org/bem/bemhint.svg)](https://travis-ci.org/bem/bemhint) [![Coverage Status](https://img.shields.io/coveralls/bem/bemhint.svg)](https://coveralls.io/r/bem/bemhint?branch=master) [![Dependency Status](https://david-dm.org/bem/bemhint.svg)](https://david-dm.org/bem/bemhint) [![devDependency Status](https://david-dm.org/bem/bemhint/dev-status.svg)](https://david-dm.org/bem/bemhint#info=devDependencies)
+
+**Can be used in projects based on [bem-bl](https://github.com/bem/bem-bl).**
```
Comment (zxqfox) [inline README.md]: `bem-bl` only?

File: README.md (hunk 1 above)
Comment (eGavr) [inline README.md]: ???

File: README.md (hunk 2 above)
Comment (eGavr) [inline README.md]: I had a task to create only such checking rule, so now the tool can be used only with `bem-bl`, probably some other rules will be implemented _tomorrow_ .

File: README.md (hunk 1 above)
Comment (zxqfox) [inline README.md]: `are wellcome.` — here. :panda_face:

**Pull request merged**